src/nvim/lua/executor.c

```diff
@@ -1779,3 +1780,58 @@ void nlua_execute_log_keystroke(int c)
 #endif
 }
 
+// Get sctx of current file being sourc3d if doesn't exsist genarate it
+static sctx_T *nlua_get_sourcing_sctx(void)
+{
+  lua_State *const lstate = global_lstate;
+  sctx_T *retval = (sctx_T *)xmalloc(sizeof(sctx_T));
+  retval->sc_seq = -1;
+  retval->sc_sid = SID_LUA;
+  retval->sc_lnum = -1;
+  lua_Debug *info = (lua_Debug *)xmalloc(sizeof(lua_Debug));
+  char *runtime_path = vim_getenv("VIMRUNTIME");
+
+#ifdef WIN32
+  // Why does $VIMRUNTIME have '/' as path separator in windows ?
```

Comment:
Because windows allows `/` as a path separator, so it's fine to always use it.

Reply:
I didn't know that . In that case we need this conversion convert this as luajit uses '\' as path sep in windows.